frontend/src/component/feature/FeatureView/FeatureOverview/ReleasePlan/hooks/useMilestoneProgressionInfo.ts

```diff
@@ -0,0 +1,36 @@
+import { formatDistanceToNow, addMinutes } from 'date-fns';
+import { useLocationSettings } from 'hooks/useLocationSettings';
+import { formatDateYMDHM } from 'utils/formatDate.ts';
+import type { MilestoneStatus } from '../ReleasePlanMilestone/ReleasePlanMilestoneStatus.tsx';
+
+export const useMilestoneProgressionInfo = (
+    intervalMinutes: number,
+    sourceMilestoneStartedAt?: string | null,
+    status?: MilestoneStatus,
```

Comment:
We need both values, because the milestone keep startedAt date, when environment is disabled.